layouts/ucf-news-card.php

```diff
@@ -36,26 +36,26 @@ function ucf_news_display_card_title( $item, $title, $display_type ) {
 }
 
 if ( ! function_exists( 'ucf_news_display_card' ) ) {
-	function ucf_news_display_card( $items, $title, $perrow, $display_type ) {
+	function ucf_news_display_card( $items, $title, $per_row, $display_type ) {
 		if ( ! is_array( $items ) ) { $items = array( $items ); }
 		ob_start();
 
-		echo '<div class="card-deck">';
+		echo '<div class="ucf-news-card-deck card-deck">';
```

Comment:
The thing I'm concerned about is: users still have the option of enabling the plugin's CSS _and_ using a theme that loads Athena.  If .card-deck or .card change at any point in the Athena framework, and those changes conflict with .ucf-news-card-deck/.ucf-news-card, we're going to have to go back and fix the conflicts in this plugin to accommodate those users.  Considering that .card-deck isn't fully responsive yet in Bootstrap 4-alpha, there's a good chance .card-deck will change in the Bootstrap beta changes that we pull into Athena (once those changes are released).

There are also some classes from Athena being used in the inner card contents (e.g. .card-title) that won't have any effect on sites not using Athena.